openapi3/example.go

```diff
@@ -55,5 +56,17 @@ func (example *Example) UnmarshalJSON(data []byte) error {
 
 // Validate returns an error if Example does not comply with the OpenAPI spec.
 func (example *Example) Validate(ctx context.Context) error {
-	return nil // TODO
+	validationOpts := getValidationOptions(ctx)
+
+	if validationOpts.ExamplesValidationDisabled {
```

Comment:
Not that kind of validation. The validation of the example against the schema should be opt-out, the validation of the example object should not.